rancher-server/variables.tf

```diff
@@ -0,0 +1,80 @@
+variable "rancher_password" {
+  type = string
+}
+
+variable "github_client_id" {
```

Comment:
Can we set a default here so that people aren't prompted to fill this out if they choose not to use github auth? Not sure off the top of my head if you can use a conditional when setting a default for a variable, but it would be nice if we could do something about that.

Reply:
Same thing for `github_client_secret`

Reply:
Defaults set to empty string. tf will error if they remain unset and gh auth is turned on, which is the best I know to do